src/os/unix.rs

```diff
@@ -225,6 +225,13 @@ pub trait WindowBuilderExt {
     fn with_resize_increments(self, increments: LogicalSize) -> WindowBuilder;
     /// Build window with base size hint. Only implemented on X11.
     fn with_base_size(self, base_size: LogicalSize) -> WindowBuilder;
+
+    /// Build window with a given app-id. It should match the `.desktop`
```

Comment:
```suggestion
    /// Build window with a given `app-id`. It should match the `.desktop`
```
Should `app-id` be in backticks?